routers/user/setting/account.go

```diff
@@ -112,7 +112,7 @@ func EmailPost(ctx *context.Context) {
 				ctx.Redirect(setting.AppSubURL + "/user/settings/account")
 				return
 			}
-			mailer.SendActivateAccountMail(ctx.Locale, ctx.User)
+			mailer.SendActivateAccountMail(ctx.Locale.Language(), ctx.User)
```

Comment:
at the moment the lang from context is used ... not sure if we should just switch to DefaultLang